kong_test.go

```diff
@@ -27,7 +27,9 @@ func (g *TestLogConsumer) Accept(l testcontainers.Log) {
 		return
 	}
 
-	g.Msgs = append(g.Msgs, string(l.Content))
+	logLine := string(l.Content)
+	g.Msgs = append(g.Msgs, logLine)
```

Comment:
We were already capturing the logs in the Msgs struct, but not doing anything with them yet.

You could print them all at the end of the test execution, or like here: print the log line, probably removing the need of storing them in the struct